src/scvi/module/_utils.py

```diff
@@ -1,45 +1,31 @@
 import torch
 
 
-def iterate(obj, func):
```

Comment:
Have you removed this function? Is it not in use at other locations?

Reply:
This is from Martin's solution.